Chatto/sources/ChatController/BaseChatViewController.swift

```diff
@@ -206,9 +199,13 @@ open class BaseChatViewController: UIViewController,
         super.viewDidLayoutSubviews()
 
         self.adjustCollectionViewInsets(shouldUpdateContentOffset: true)
-        self.keyboardTracker.adjustTrackingViewSizeIfNeeded()
+        self.keyboardUpdatesHandler.adjustLayoutIfNeeded()
 
         self.updateInputContainerBottomBaseOffset()
+
+        self.viewEventsHandlers.forEach {
+            $0.onViewDidLayoutSubviews()
+        }
```

Comment:
Why do we need to call onViewDidLayoutSubviews on viewWillAppear?

Reply:
It is not, github is collapsing some lines :)